api/triage_metadata.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"context"
+	"errors"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/google/go-github/v28/github"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	sourceOwner   = "web-platform-tests"
+	sourceRepo    = "wpt-metadata"
+	commitMessage = "Commit New Metadata"
+	commitBranch  = "auto-triage-branch" + generateRandomInt()
```

Comment:
Good catch! The purpose of `generateRandomInt()` is to help create a new checkout from master branch every time. 

It is a bug because I didn't consider the lifecycle of a package-level global variables. Global variables should be generally avoided due to concurrent issues, unless its ready-only. I can make all of them constant variables besides commitBranch, but the neater way is probably to define a struct, turn them into local variables and pass it along.